src/main/java/com/ietswise/service/impl/GoogleEventsServiceImpl.java

```diff
@@ -98,4 +109,148 @@ private ZonedDateTime extractDate(JSONObject dateTime) {
         }
         return null;
     }
+
+    /**
+     * Method for getting all free and busy hours of all days of the month
+     * @param tutorId tutor's email
+     * @param year year
+     * @param month month number
+     * @return List<FreeAndBusyHoursOfTheDay>
+     */
+    @Override
+    public List<FreeAndBusyHoursOfTheDay> getEventsByYearAndMonth(String tutorId, int year, int month) {
+        try {
+            ZonedDateTime startOfMonth = YearMonth.of(year, month).atDay(1).atStartOfDay(ZoneId.systemDefault());
+            ZonedDateTime endOfMonth = YearMonth.of(year, month).atEndOfMonth().atStartOfDay(ZoneId.systemDefault());
```

Comment:
The length of the line in our project should not exceed 120 characters. Don't forget about static imports to make your code more readable.